src/pages/preview.vue

```diff
@@ -0,0 +1,398 @@
+          ><template #icon>
+            <DecoratedIcon iconName="room"></DecoratedIcon></template
+        ></CourseDetail>
+
+        <CourseDetail item="授業形式" value="対面"
+          ><template #icon>
+            <DecoratedIcon iconName="category"></DecoratedIcon></template
+        ></CourseDetail>
+      </div>
+    </section>
+
+    <section class="preview__content--flex">
+      <section class="preview__content">
+        <Sidebar :isLogin="true"></Sidebar>
+      </section>
+      <section class="preview__content">
+        <SidebarContent
+          @click="displayLog"
+          iconName="home"
+          item="ホーム"
+        ></SidebarContent>
+        <SidebarContent
+          @click="displayLog"
+          :link="true"
+          iconName="home"
+          item="ホーム"
+        ></SidebarContent>
+        <SidebarContent
+          @click="displayLog"
+          iconName="home"
+          item="ホーム"
+          :selected="true"
+        ></SidebarContent>
+      </section>
+    </section>
+    <section>
+      <div class="card-add-wrapper">
+        <CardAdd
+          @click-next-button="displayLog"
+          iconName="search"
+          heading="授業の検索"
+          text="ワードや条件を指定して授業を検索・追加します。"
+        ></CardAdd>
+      </div>
+
+      <div class="card-course-wrapper">
+        <CardCourse
+          @click-checkbox="isCourseCheked = !isCourseCheked"
+          @click-syllabus-link="displayLog"
+          :isChecked="isCourseCheked"
+          :course="courseInfo"
+        ></CardCourse>
+      </div>
+    </section>
+    <div class="preview__content">
+      <Popup>
+        <PopupContent
+          v-for="data in popupData"
+          :key="data.value"
+          @click="data.onClick"
+          :link="data.link"
+          :value="data.value"
+          :color="data.color"
+        ></PopupContent>
+      </Popup>
+    </div>
+    <div class="preview__content">
+      <Popup>
+        <PopupContent
+          v-for="data in popupModuleData"
+          :key="data.value"
+          @click="data.onClick"
+          :value="data.value"
+        >
+        </PopupContent>
+      </Popup>
+    </div>
+    <section class="preview__content">
+      <Button @click="openModal" size="small">Open Modal</Button>
+    </section>
+    <section class="preview__content">
+      <Modal v-if="modal" @click="closeModal" size="small">
+        <template #title>title</template>
+        <template #contents>
+          <div class="contents">contents</div>
+        </template>
+        <template #button>
+          <Button @click="closeModal" layout="fill" size="medium"
+            >Button</Button
+          >
+          <Button
+            @click="clickHandler"
+            layout="fill"
+            size="medium"
+            color="primary"
+            >Button</Button
+          >
+        </template>
+      </Modal>
+    </section>
+    <PageHeader
+      :atHome="true"
+      :calendar="{ month: 2, day: 23, week: '火', schedule: '通常日課' }"
+    >
```

Comment:
```suggestion
    <PageHeader
      atHome
      :calendar="{ month: 2, day: 23, week: '火', schedule: '通常日課' }"
    >
```
booleanはこの記法が直感的
falseなら記述しない